extensions/amp-dailymotion/amp-dailymotion.md

```diff
@@ -110,6 +110,16 @@ Indicates whether to show video information (title and owner) on the start scree
 * Value: `"true"` or `"false"`
 * Default value: `"true"`
 
+**data-param-*** (optional)
+
+All `data-param-*` attributes will be added as query parameter to the Dailymotion iframe src. This may be used to pass custom values not explicitly declared.
```

Comment:
Rewrite as: 

All `data-param-*` attributes are added as query parameters to the `src` value of the embedded Dailymotion iframe. You can use this attribute to pass custom values not explicitly declared.